google/google-how-to/gce/gce-image-retention-policy.rst

```diff
@@ -0,0 +1,35 @@
+GCE Image Retention Policies
+=========================
+
+GCE Ubuntu Image Deletion
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Ubuntu cloud images have a concept of "daily" and "release" images. In a nutshell, daily images are untested builds with cutting edge features and package versions.
+Release images are tested and therefore benefit from Canonical's in-life support. Please see how to list both "daily" and "release" images here: :doc:`find-ubuntu-images`.
+
+As of late 2023, older daily and release images are subject to a retention policy. The policy can be summarised as follows:
```

Comment:
Let's not put in a date, as it seems to imply that we won't be keeping our docs current. It is a safer and more accurate statement, but it won't build trust. So we'll just have try to ensure that we make relevant changes to this doc whenever the retention policy changes the next time.